test.js

```diff
@@ -1,13 +1,16 @@
-(helloCatAsync(function(result) {
-    console.log(result);
-    test = result;
-}));
+class A {
+    foo() {
```

Comment:
You have a terrible habit of adding random files, why proof read your stuff?

Reply:
you're right